src/leveled_sst.erl

```diff
@@ -843,16 +842,6 @@ reader(close, _From, State) ->
     ok = file:close(State#state.handle),
     {stop, normal, ok, State}.
 
-reader(timeout, State) ->
```

Comment:
I am very surprised you can delete this timeout handling!

There are still a DELETE_TIMEOUT and STARTUP_TIMEOUT that would kick in now and then and that would cause this code to execute. So why does this work when no timeout handling in reader any more?